pkg/stub/handler.go

```diff
@@ -169,7 +195,15 @@ func (h *Handler) manageDockerCfgSecret(deleted bool, samplesResource *v1alpha1.
 		}
 		logrus.Printf("registry dockerconfig secret %s was deleted", secret.Name)
 		newStatus = corev1.ConditionFalse
+		h.registrySecret = nil
 	} else {
+		if h.registrySecret != nil {
+			currentVersion, _ := strconv.Atoi(h.registrySecret.ResourceVersion)
+			newVersion, _ := strconv.Atoi(secret.ResourceVersion)
+			if newVersion <= currentVersion {
```

Comment:
when would new version be less than current version?

Reply:
when a paranoid developer thought he saw such an event, albeit very intermittently, during a flurry of testing on top of all this new stuff 

this is definitely erring on the side of caution / paranoia

Reply:
ok